Sources/SwiftInspectorCore/ImportsAnalyzer.swift

```diff
@@ -97,7 +97,7 @@ public final class ImportsAnalyzer: Analyzer {
     var submoduleIdentifier: String = ""
 
     for child in syntaxNode.children {
-      guard let accessPath = child as? AccessPathSyntax else {
+      guard let accessPath = child.as(AccessPathSyntax.self) else {
```

Comment:
Can you explain this to me?

Reply:
@bachand how Swiftsyntax structures nodes moved from being protocol based to struct based. You can read more about [this here](https://github.com/apple/swift-syntax/blob/master/Changelog.md#modelling-syntax-nodes-as-structs)